### PR TITLE
Issue #2606 Doc - Start libvirtd service if not running

### PR DESCRIPTION
**Pull request by claudio4j**

### CODE REVIEW

**Comment by praveenkumar** [comment]
add to whitelist.

**Comment by praveenkumar** [comment]
retest this please.

File: docs/source/getting-started/setting-up-virtualization-environment.adoc [hunk 1]
```diff
@@ -111,6 +111,14 @@ $ sudo sed -ri 's/.?group\s?=\s?".+"/group = "kvm"/1' /etc/libvirt/qemu.conf
 $ newgrp libvirt
 ----
 
+=== Start libvirtd service
+
+Start *libvirtd* service if not running.
+
+----
+$ sudo systemctl start libvirtd
```
Comment (robin-owen) [inline docs/source/getting-started/setting-up-virtualization-environment.adoc]: I'd recommend checking the status of **libvirtd** first via `systemctl is-active libvirtd`. If **libvirtd** is not active, start the service via `sudo systemctl start libvirtd`:

```adoc
. Check the status of **libvirtd**:
+
----
$ systemctl is-active libvirtd
----

. If **libvirtd** is not active, start the **libvirtd** service:
+
----
$ sudo systemctl start libvirtd
----
```

Comment (claudio4j) [reply docs/source/getting-started/setting-up-virtualization-environment.adoc]: Integrated your requested change, thanks.

File: docs/source/getting-started/setting-up-virtualization-environment.adoc [hunk 2]
```diff
@@ -111,6 +111,20 @@ $ sudo sed -ri 's/.?group\s?=\s?".+"/group = "kvm"/1' /etc/libvirt/qemu.conf
 $ newgrp libvirt
 ----
 
+=== Start libvirtd service
```
Comment (robin-owen) [inline docs/source/getting-started/setting-up-virtualization-environment.adoc]: Apologies, I initially overlooked that we also need an anchor ID here:

```adoc
[[kvm-driver-libvirtd-service]]
=== Start libvirtd service
```

Comment (claudio4j) [reply docs/source/getting-started/setting-up-virtualization-environment.adoc]: Done.

**Pull request merged**